lib/teleterm/daemon/daemon.go

```diff
@@ -32,11 +32,13 @@ import (
 
 	"github.com/gravitational/teleport/api/client/proto"
 	accesslistv1 "github.com/gravitational/teleport/api/gen/proto/go/teleport/accesslist/v1"
+	devicetrustv1 "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
```

Comment:
Suggestion: I tend to use `devicepb`:

```suggestion
	devicepb "github.com/gravitational/teleport/api/gen/proto/go/teleport/devicetrust/v1"
```